projects/1.16/assets/macaws-roofs/mcwroofs/lang/zh_cn.json

```
@@ -1 +1,609 @@
+  "block.mcwroofs.light_gray_top_roof":"淡灰色顶端屋顶",
+  "block.mcwroofs.light_gray_three_way_roof":"淡灰色三向屋顶",
+  "block.mcwroofs.light_gray_four_way_roof":"淡灰色四向屋顶",
+  "block.mcwroofs.light_gray_lower_roof":"淡灰色低屋顶",
+  "block.mcwroofs.light_gray_steep_roof":"淡灰色陡峭屋顶",
+  "block.mcwroofs.gray_roof":"灰色屋顶",
+  "block.mcwroofs.gray_attic_roof":"灰色天窗屋顶",
+  "block.mcwroofs.gray_cube_roof":"灰色方块屋顶",
+  "block.mcwroofs.gray_top_roof":"灰色顶端屋顶",
+  "block.mcwroofs.gray_three_way_roof":"灰色三向屋顶",
+  "block.mcwroofs.gray_four_way_roof":"灰色四向屋顶",
+  "block.mcwroofs.gray_lower_roof":"灰色低屋顶",
+  "block.mcwroofs.gray_steep_roof":"灰色陡峭屋顶",
+  "block.mcwroofs.black_roof":"黑色屋顶",
+  "block.mcwroofs.black_attic_roof":"黑色天窗屋顶",
+  "block.mcwroofs.black_cube_roof":"黑色方块屋顶",
+  "block.mcwroofs.black_top_roof":"黑色顶端屋顶",
+  "block.mcwroofs.black_three_way_roof":"黑色三向屋顶",
+  "block.mcwroofs.black_four_way_roof":"黑色四向屋顶",
+  "block.mcwroofs.black_lower_roof":"黑色低屋顶",
+  "block.mcwroofs.black_steep_roof":"黑色陡峭屋顶",
+  "block.mcwroofs.base_roof":"基本款屋顶",
+  "block.mcwroofs.base_attic_roof":"基本款天窗屋顶",
+  "block.mcwroofs.base_cube_roof":"基本款方块屋顶",
+  "block.mcwroofs.base_top_roof":"基本款顶端屋顶",
+  "block.mcwroofs.base_three_way_roof":"基本款三向屋顶",
+  "block.mcwroofs.base_four_way_roof":"基本款四向屋顶",
+  "block.mcwroofs.base_lower_roof":"基本款低屋顶",
+  "block.mcwroofs.base_steep_roof":"基本款陡峭屋顶",
+
+  "block.mcwroofs.stone_roof":"石屋顶",
+  "block.mcwroofs.stone_attic_roof":"石天窗屋顶",
+  "block.mcwroofs.stone_cube_roof":"石方块屋顶",
+  "block.mcwroofs.stone_top_roof":"石顶端屋顶",
+  "block.mcwroofs.stone_three_way_roof":"石三向屋顶",
+  "block.mcwroofs.stone_four_way_roof":"石四向屋顶",
+  "block.mcwroofs.stone_lower_roof":"石低屋顶",
+  "block.mcwroofs.stone_steep_roof":"石陡峭屋顶",
+  "block.mcwroofs.granite_roof":"花岗岩屋顶",
+  "block.mcwroofs.granite_attic_roof":"花岗岩天窗屋顶",
+  "block.mcwroofs.granite_cube_roof":"花岗岩方块屋顶",
+  "block.mcwroofs.granite_top_roof":"花岗岩顶端屋顶",
+  "block.mcwroofs.granite_three_way_roof":"花岗岩三向屋顶",
+  "block.mcwroofs.granite_four_way_roof":"花岗岩四向屋顶",
+  "block.mcwroofs.granite_lower_roof":"花岗岩低屋顶",
+  "block.mcwroofs.granite_steep_roof":"花岗岩陡峭屋顶",
+  "block.mcwroofs.diorite_roof":"闪长岩屋顶",
+  "block.mcwroofs.diorite_attic_roof":"闪长岩天窗屋顶",
+  "block.mcwroofs.diorite_cube_roof":"闪长岩方块屋顶",
+  "block.mcwroofs.diorite_top_roof":"闪长岩顶端屋顶",
+  "block.mcwroofs.diorite_three_way_roof":"闪长岩三向屋顶",
+  "block.mcwroofs.diorite_four_way_roof":"闪长岩四向屋顶",
+  "block.mcwroofs.diorite_lower_roof":"闪长岩低屋顶",
+  "block.mcwroofs.diorite_steep_roof":"闪长岩陡峭屋顶",
+  "block.mcwroofs.andesite_roof":"安山岩屋顶",
+  "block.mcwroofs.andesite_attic_roof":"安山岩天窗屋顶",
+  "block.mcwroofs.andesite_cube_roof":"安山岩方块屋顶",
+  "block.mcwroofs.andesite_top_roof":"安山岩顶端屋顶",
+  "block.mcwroofs.andesite_three_way_roof":"安山岩三向屋顶",
+  "block.mcwroofs.andesite_four_way_roof":"安山岩四向屋顶",
+  "block.mcwroofs.andesite_lower_roof":"安山岩低屋顶",
+  "block.mcwroofs.andesite_steep_roof":"安山岩陡峭屋顶",
+  "block.mcwroofs.cobblestone_roof":"原石屋顶",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "block.mcwroofs.cobblestone_roof":"圆石屋顶",
```

---

已修正